redcap_bridge/project_building.py

```diff
@@ -160,6 +166,17 @@ def combine_values(value1, value2):
     if output_file is not None:
         combined_df.to_csv(output_file, index=False)
 
+    # print number of customizing field
+    print("\nCustomizing", len(customization_df) , "fields\n", end='')
+
+    # print the field name
+    if(len(customization_df)) < 100:
+        customization_df = pd.read_csv(customization_csv, usecols=[0])
+        print(customization_df)
+    else:
+        print_custom.append(customization_df['Variable / Field Name'])
```

Comment:
Why do you use the `print_custom` in the `else` case? It doesn't seem like this variable is used for anything else earlier. Maybe removing it would be a cleaner approach?